test/common/testlib.py

```diff
@@ -1634,7 +1634,7 @@ def new_browser(self, machine: testvm.Machine | None = None, coverage: bool = Fa
                 pass
             else:
                 if machine.image == reference_image:
-                    pixels_label = self.label()
+                    pixels_label = self.label() + ('-efi' if machine.is_efi else '')
```

Comment:
mypy rightfully complains here:
```
not ok 6 /static-code/test-mypy
# test/common/testlib.py:1637: error: "Machine" has no attribute "is_efi"  [attr-defined]
```

This doesn't work as this is only a valid attribute for `VirtMachine` instances, but testlib works with the more general `Machine` class (which could also be an existing `SSHConnection` with `--machine`).

This also feels overly generalized -- most pixel tests shouldn't care at all if the machine is EFI or not. It would mean that a lot of/most pixel refs had to be duplicated when introducing an EFI test variant. We don't want that.

Wouldn't it be more prudent to modify `b.pixels_label` in your anaconda test class?

Reply:
True. I can do that it's just 1 case affected at this point.

Reply:
I will remove this pixel test for now and resolve this in a follow up from my EFI refactor.